yt/visualization/base_plot_types.py

```diff
@@ -157,8 +157,9 @@ def _get_canvas_classes(self):
 
         try:
             module, fig_canvas, fig_manager = BACKEND_SPECS[key]
-        except KeyError:
-            return
+        except KeyError as err:
+            msg = f"{key} is not in list of valid backends: {list(BACKEND_SPECS)}."
```

Comment:
i know i just pushed this up, but i'm now thinking i **should** just remove the whole try-except, the default keyerror message that would print is probably sufficient. but i'll let tests finish running...